pandas/tests/series/test_analytics.py

```diff
@@ -1788,3 +1788,11 @@ def test_value_counts_categorical_not_ordered(self):
                         index=exp_idx, name='xxx')
         tm.assert_series_equal(s.value_counts(normalize=True), exp)
         tm.assert_series_equal(idx.value_counts(normalize=True), exp)
+
+    def test_nat_operations(self):
```

Comment:
OK, I see that `TestTimeSeries` mixes tests for both cases (dt index or dt values), so will put it there.

(although I would find it more logical to separate those)